pkg/storage/unified/resource/server.go

```diff
@@ -122,6 +127,18 @@ type StorageBackend interface {
 	GetResourceStats(ctx context.Context, namespace string, minCount int) ([]ResourceStats, error)
 }
 
+type ModifiedResourceKey struct {
```

Comment:
Can we do with `resourcepb.ResourceKey` here?

Reply:
yep 👍